modules/python/src2/cv2.cpp

```diff
@@ -206,6 +206,23 @@ bool pyopencv_to(PyObject* obj, T& p, const char* name = "<unknown>");
 template<typename T> static
 PyObject* pyopencv_from(const T& src);
 
+template <typename T> static
+bool pyopencv_to_ptr(PyObject *o, T* p, const char *name = "<unknown>")
+{
+    if (!o || o == Py_None)
+        return true;
+    p = new T();
```

Comment:
Where will this memory be freed? Could you please add a test for the new functionality?